charts/gateway-addons-helm/values.yaml

```diff
@@ -10,6 +11,38 @@ grafana:
   adminPassword: admin
   service:
     type: LoadBalancer
+  dashboardProviders:
+    dashboardproviders.yaml:
+      apiVersion: 1
+      providers:
+        - name: 'envoy-gateway'
+          orgId: 1
+          folder: 'envoy-gateway'
+          type: file
+          disableDeletion: false
+          editable: true
+          options:
+            path: /var/lib/grafana/dashboards/envoy-gateway
```

Comment:
I guess the dashboards in the followings configuration will be downloaded and placed in this directory by grafana?

Reply:
Oh, it's not.